content/en/blog/_posts/2022-09-14-pod-has-network-condition.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: blog
-title: 'Kubernetes 1.25: PodHasNetwork Condition for Pods'
+title: 'Kubernetes 1.25: PodReadyToStartContainers Condition for Pods'
```

Comment:
History is about something happened, we are not supposed to rewrite history, for whatever reason.

It is okay to add a subsection or a callout that this feature gate has been renamed starting 1.28.

Reply:
I added it to the pod docs.  Are you saying I should just add a section to this post saying that PodHasNetwork -> PodReadyToStartContainers?

Reply:
Either a new section or a new page sounds good to me.

Reply:
updated with just a section saying that we renamed it.